sendcodes.py

```diff
@@ -101,11 +103,14 @@ def get_random_base64_string():
         # Parse the date.
         parsed_date = datetime.strptime(row[REPORTED_DATE], '%m/%d/%Y')
 
+        # calculate offset based on testDate to handle daylight savings time
```

Comment:
So far there's an undocumented convention of making comments sentences (i.e. caps + punctuation). What do you think about any style guide for this file? Are you aware of one we could easily adopt?

Reply:
No problem